### ISSUE TITLE
Experiments

### ISSUE THREAD

**Issue by psaegert** [MrWhatZitToYaa/IGNNS-final-project#35]
## Hyperparameter Optimization
![image](https://github.com/MrWhatZitToYaa/IGNNS-final-project/assets/36567814/9714ee68-da2a-4184-b19f-e63be0a8fe0c)

**Stage 1 (Random Init & GP Hedge)**
Stuck in local minimum

**Stage 2 (Halton Init & Expected Improvement)**
- Best Model ("small"), val loss: -61.49 ± 0.25
```yaml
model:
  kwargs:
    size: 21
    nested_sizes: [16, 16, 16, 16, 16, 16, 16]
    n_conditions: 80
    n_blocks: 32
    dropout: 0.383
    act_norm: True
```

- 2nd-Best Model ("large"), val loss: -61.3 ± 2.0
```yaml
model:
  kwargs:
    size: 19
    nested_sizes: [526, 526, 526, 526, 526]
    n_conditions: 1360
    n_blocks: 26
    dropout: 0.407
    act_norm: True
```

## Trajectories
- [x] `trajectory_FC_small`: (-,  FC) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/xfkfo69
- [x] `trajectory_LSTM_small`: (-, LSTM) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/1qwwz99a
- [x] `trajectory_TRF_small`: (-, Transformer) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/m8fqmi3n
- [x] `trajectory_FC_large`: (-,  FC) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/v83m23td
- [x] `trajectory_LSTM_large`: (-, LSTM) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/eu00x1d2
- [x] `trajectory_TRF_large`: (-, Transformer) $\rightarrow$ https://wandb.ai/balisticcnf/bcnf-test/runs/tia1qixl

## Videos
- [ ] `video_LSTM_large`: (ConvNet, LSTM) $\rightarrow$ [worldly-silence-93](https://wandb.ai/balisticcnf/bcnf-test/runs/6p2xv2wl)
